<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPacketDialog</class>
 <widget class="QDialog" name="NewPacketDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="newPacketDlgButtonBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabPacketType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>631</width>
     <height>361</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabTCP">
    <attribute name="title">
     <string>TCP</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabUDP">
    <attribute name="title">
     <string>UDP</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabIP">
    <attribute name="title">
     <string>IP</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabARP">
    <attribute name="title">
     <string>ARP</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabICMP">
    <attribute name="title">
     <string>ICMP</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QComboBox" name="ipModeComboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>390</y>
     <width>104</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>IPv4 Mode</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>IPv6 Mode</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>IP Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>392</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Capture</string>
   </property>
  </widget>
  <widget class="QComboBox" name="captureComboBox">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>390</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>&lt;unseleted&gt;</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>422</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Source IP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Destination IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="srcIPEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="dstIPEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>450</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>newPacketDlgButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewPacketDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
